docker-app/qfieldcloud/core/views/package_views.py

```diff
@@ -224,3 +249,228 @@ def post(self, request, project_id, job_id, filename):
                 "md5sum": md5sum,
             }
         )
+
+
+@extend_schema_view(
+    get=extend_schema(
+        description="Get all the files in a project package with files.",
+        responses={200: LatestPackageSerializer()},
+    ),
+)
+class LatestPackageView(views.APIView):
+    permission_classes = [permissions.IsAuthenticated, PackageViewPermissions]
+
+    def get(self, request, project_id):
+        """Get last project package status and file list."""
+        project = Project.objects.get(id=project_id)
```

Comment:
```suggestion
        project = get_object_or_404(Project, id=project_id)
```

This seem to be used in the view just below